src/common/HeartbeatMap.cc

```diff
@@ -71,13 +71,13 @@ bool HeartbeatMap::_check(const heartbeat_handle_d *h, const char *who, time_t n
 
   was = h->timeout;
   if (was && was < now) {
-    ldout(m_cct, 1) << who << " '" << h->name << "'"
+    ldout(m_cct, 0) << who << " '" << h->name << "'"
```

Comment:
This one is really not a great idea to bump up; at level zero there's not much in the logs and while exceeding the expected time is *bad* it's not worth being the only log-spammer for either.